app/services/github_handler.rb

```diff
@@ -0,0 +1,29 @@
+class GithubHandler
+  def initialize (event, payload)
+    @event = event
+    @payload = payload
+  end
+
+  def handle
```

Comment:
are we validating that the request comes actually from github? is there a ticket to add that in the future?